metadata/db-xrefs.yaml

```diff
@@ -2373,7 +2373,7 @@
   name: Saccharomyces Genome Database
   synonyms:
     - SGDID
-  rdf_uri_prefix: https://identifiers.org/sgd
+  rdf_uri_prefix: http://identifiers.org/sgd
```

Comment:
@sierra-moxon this change looks good but it seems like this namespace should also have a trailing slash in order for the URI construction to be correct.

Reply:
```suggestion
  rdf_uri_prefix: http://identifiers.org/sgd/
```